### PR TITLE
migrate product_attribute_types addon from odoomrp-wip

### PR DESCRIPTION
**Pull request by rdeheele**
@pedrobaeza @max3903 @anajuaristi
This PR proposes to add the addon created here https://github.com/odoomrp/odoomrp-wip

Description
"This module extends product attributes, adds new type range, also the option of
defining if it is required and a default value for the attribute."

It uses the migration method explained here https://github.com/odoomrp/odoomrp-wip/wiki/Migration-to-OCA to keep commits history.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Product attribute types
```
Comment (pedrobaeza) [inline README.rst:1]: Put AGPL-3 badge

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Credits
+=======
+
+Contributors
+------------
+* Mikel Arregi <mikelarregi@avanzosc.es>
+* Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>
+* Pedro M. Baeza <pedro.baeza@serviciobaeza.com>
+* Ana Juaristi <ajuaristio@gmail.com>
```
Comment (pedrobaeza) [inline README.rst:16]: Put OCA as maintainer

File: __openerp__.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+{
+    "name": "Product attribute types",
+    "version": "1.0",
+    "depends": [
+        "product",
+    ],
+    "author": "OdooMRP team,"
+              "AvanzOSC,"
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza",
```
Comment (pedrobaeza) [inline __openerp__.py:27]: Put "Odoo Community Association (OCA)" as author

File: models/product.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+
+class ProductAttributeValue(models.Model):
+    _inherit = "product.attribute.value"
+
+    attr_type = fields.Selection(string='Type',
+                                 related='attribute_id.attr_type')
+    numeric_value = fields.Float('Numeric Value', digits=(12, 6))
+    min_range = fields.Float('Min', digits=(12, 6))
+    max_range = fields.Float('Max', digits=(12, 6))
```
Comment (pedrobaeza) [inline models/product.py:47]: Now that I have see it again, it lacks a constraint that check min_range < max_range

File: README.rst [hunk 5]
```diff
@@ -0,0 +1,16 @@
+Product attribute types
+=======================
+
+This module extends product attributes, adds new type range, also the option of
```
Comment (pedrobaeza) [inline README.rst:4]: s/adds new/adding a new

Comment (pedrobaeza) [inline README.rst:4]: s/also the/and also the

**Comment by rdeheele** [comment]
@pedrobaeza I close this PR because there is a directory problem with the history rewriting. I will create a new one.

**Comment by pedrobaeza** [comment]
OK, no problem. Please discuss also if we rename the module to a singular form.

**Pull request closed without merging**